lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -1180,6 +1181,61 @@ describe('workers/repository/process/lookup/index', () => {
       );
     });
 
+    it('should warn if no digest could be found but there is a current digest', async () => {
+      config.currentValue = 'v1.0.0';
+      config.currentDigest = 'bla';
+      config.digestOneAndOnly = true;
+      config.depName = 'angular/angular';
+      config.datasource = GithubTagsDatasource.id;
+
+      // Only mock calls once so that the second invocation results in
+      // no digest being computable.
+      jest.spyOn(githubGraphql, 'queryReleases').mockResolvedValueOnce([]);
```

Comment:
please only one spyOn otherwise this won't work with next jest minor version

Reply:
can you please explain more? I don't know much about Jest. what do you prefer here?

Reply:
if you call spyon two times for the same function, jest will create a spy function over a spy function. this can cause test issues and will be an error in next jest minor which is already blocked by other tests in this repo. so we need to fix them all to successfully update jest.

I'll make a new suggestions to show you the right way.

Reply:
ah, sorry. i mixed something. leave code as is.